### PR TITLE
Updated English C courses

### PR DESCRIPTION
**Pull request by apu52**
## What does this PR do?
Add resource(s) 

### Description
I have improved English sources repo. I have added some English C programming courses.
### Why is this valuable (or not)?
This are very valuable courses
### How do we know it's really free?
It's totally free
### For book lists, is it a book? For course lists, is it a course? etc.
It's a course list
## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by Ajay-singh1** [comment]
LGTM!

**Review by Ajay-singh1 — APPROVED** [review]
LGTM!

**Comment by apu52** [comment]
@eshellman Please merge this PR

**Comment by eshellman** [comment]
checking for build problems

File: courses/free-courses-hi.md
Comment (eshellman) [inline courses/free-courses-hi.md:155]: duplicate

**Comment by apu52** [comment]
@eshellman I have removed the duplicate link & also have added C programming Course in English .Please merge it

**Review by Ajay-singh1 — COMMENTED** [review]
Wait for the maintainer's response

**Comment by apu52** [comment]
> There is nothing new here you just added a duplicate course and removed it.I think this is an invalid PR.

I have added a F programming course in the English course section....please check it

**Comment by apu52** [comment]
This course....I have added & also deleted the duplicate course which was not added by me

**Comment by apu52** [comment]
@Ajay-singh1 @eshellman

**Comment by Ajay-singh1** [comment]
> @Ajay-singh1 @eshellman

Wait for Eric.

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**
